app/models/miq_server/worker_management/monitor.rb

```diff
@@ -26,7 +26,18 @@ def monitor_workers
     processed_worker_ids += check_not_responding
     processed_worker_ids += check_pending_stop
     processed_worker_ids += clean_worker_records
-    processed_worker_ids += request_workers_to_sync_config(resync_needed)
+
+    # Monitor all remaining current worker records
+    processed_worker_ids += miq_workers.where(:status => MiqWorker::STATUSES_CURRENT_OR_STARTING).each do |worker|
+      # Check their queue messages for timeout
+      worker.validate_active_messages
```

Comment:
This is going away in the other PR, rights?  Or did you want this PR in first?

Reply:
No this is the method on an individual worker. I didn't actually change this one.

This is actually why the workers we process here are added to the processed worker list because these are the handlers we skip in the call below.

That said, I could probably remove this one as well in the follow-up.